dashboard/DashboardDB.py

```diff
@@ -208,6 +222,77 @@ def get_squeue_entries(self, hpc: const.HPC):
 
         return [SQueueEntry(*result) for result in results]
 
+    def update_daily_quota(self, entries: Iterable[QuotaEntry], hpc: const.HPC):
+        table = self.get_quota_t_name(hpc)
+        day = date.today()
+
+        with self.get_cursor(self.db_file) as cursor:
+            for ix, entry in enumerate(entries):
+                result = cursor.execute(
+                    "SELECT * FROM {} WHERE FILE_SYSTEM = ? AND DAY = ?".format(table),
+                    (entry.file_system, day),
+                ).fetchone()
+
+                if result:
+                    cursor.execute(
+                        "UPDATE {} SET "
+                        "USED_SPACE = ?, AVAILABLE_INODES = ?, USED_INODES = ? WHERE FILE_SYSTEM = ? AND DAY = ?".format(
+                            table
+                        ),
+                        (
+                            entry.used_space,
+                            entry.available_inodes,
+                            entry.used_inodes,
+                            entry.file_system,
+                            entry.day,
+                        ),
+                    )
+                else:
+                    cursor.execute(
+                        "INSERT INTO {} VALUES (?, ?, ?, ?, ?)".format(table),
+                        (
+                            entry.file_system,
+                            entry.used_space,
+                            entry.available_inodes,
+                            entry.used_inodes,
+                            entry.day,
+                        ),
+                    )
+
+    def get_daily_inodes(self, hpc: const.HPC, file_system="nobackup"):
+        """
+        Get inodes usage for a particular file system eg.nobackup/project
+        """
+        sql = "SELECT FILE_SYSTEM, USED_INODES, DAY FROM {} WHERE FILE_SYSTEM LIKE ?".format(
+            self.get_quota_t_name(hpc)
+        )
+        with self.get_cursor(self.db_file) as cursor:
+            results = cursor.execute(sql, ("%{}%".format(file_system),)).fetchall()
+        return results
+
+    def get_daily_quota(
+        self,
+        hpc: const.HPC,
+        day: Union[date, str] = None,
+        file_system="nobackup",
+    ):
+        """
+        Get daily quota usage for a particular file system eg.nobackup/project
```

Comment:
Single line docstrings should be of the format 
"""Single line Docstring"""  

and multi line docstrings
"""First line
Second line
"""